README.md

```diff
@@ -80,7 +80,7 @@ of VNFs based on the VPP Agent:
 ## Quickstart
 
 For a quick start with the VPP Agent, you can use pre-built Docker images with
-the Agent and VPP on [Dockerhub][14].
+the Agent and VPP on Dockerhub [for X86_64 architecture][14] and [for ARM64 architecture][17].
```

Comment:
The correct name is `x86_64`, without uppercase `x`.

Reply:
Done.